tests/upgrades/test_contentview.py

```diff
@@ -279,3 +299,178 @@ def test_post_user_scenario_bug_1429201(self):
         self.assertIsNotNone(container_ids)
         self.assertIn('Error', list(result_fail.values())[0])
         self.assertIn('Complete', list(result_pass.values())[0])
+
+
+class CVUpgradeScenario(CLITestCase):
+    """ Verify if we can update Content-view post-upgrade
```

Comment:
please create the doc string as : https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_remoteexecution.py#L29